mh/constants.py

```diff
@@ -177,13 +177,13 @@ def make_binary_npc_greeters(is_jap=False):
         event_quests = b""
 
     event_quests += b"\n".join([
-        get_quest_name(QUEST_EVENT_JUMP_FOUR_JAGGI),
-        get_quest_name(QUEST_EVENT_BLOOD_SPORT),
-        get_quest_name(QUEST_EVENT_MERCY_MISSION),
-        get_quest_name(QUEST_EVENT_THE_PHANTOM_URAGAAN),
-        get_quest_name(QUEST_EVENT_WORLD_EATER),
-        get_quest_name(QUEST_EVENT_WHERE_GODS_FEAR_TO_TREAD),
-        get_quest_name(QUEST_EVENT_GREEN_EGGS)
+        QUEST_EVENT_JUMP_FOUR_JAGGI['quest_info']['name'],
+        QUEST_EVENT_BLOOD_SPORT['quest_info']['name'],
+        QUEST_EVENT_MERCY_MISSION['quest_info']['name'],
+        QUEST_EVENT_THE_PHANTOM_URAGAAN['quest_info']['name'],
+        QUEST_EVENT_WORLD_EATER['quest_info']['name'],
+        QUEST_EVENT_WHERE_GODS_FEAR_TO_TREAD['quest_info']['name'],
+        QUEST_EVENT_GREEN_EGGS['quest_info']['name']
```

Comment:
This raises an exception due to `bytes` being mixed up with `str`. Here is one way to get rid of it:
```python
QUEST_LIST = (
    QUEST_EVENT_JUMP_FOUR_JAGGI,
    QUEST_EVENT_BLOOD_SPORT,
    QUEST_EVENT_MERCY_MISSION,
    QUEST_EVENT_THE_PHANTOM_URAGAAN,
    QUEST_EVENT_WORLD_EATER,
    QUEST_EVENT_WHERE_GODS_FEAR_TO_TREAD,
    QUEST_EVENT_GREEN_EGGS
)
event_quests += b"\n".join([
    quest['quest_info']['name'].encode("ascii")
    for quest in QUEST_LIST
])
```